### ISSUE TITLE
Dual Blades "Holy Sabers" rarity is incorrect

### ISSUE THREAD

**Issue by rjb8682** [LartTyler/MHWDB-Docs#8]
The rarity on the Holy Sabers is coming through as 252. Here's is the doc for that weapon:

```
{
    "id":291,
    "slug":"holy-sabers",
    "name":"Holy Sabers",
    "type":"dual-blades",
    "rarity":252,
    "attributes":{
        "attack":252,
        "slotsRank3":1,
        "elementDamage":120,
        "elementType":"Water",
        "sharpnessRed":15,
        "sharpnessOrange":12,
        "sharpnessYellow":15,
        "sharpnessGreen":30,
        "sharpnessBlue":15
    },
    "crafting":{
        "craftable":false,
        "previous":290,
        "branches":[

        ],
        "craftingMaterials":[

        ],
        "upgradeMaterials":[
            {
                "quantity":6,
                "item":{
                    "id":123,
                    "name":"Dragonvein Crystal",
                    "description":"Mined from high rank areas. Many expect this newly-discovered crystal to usher in a new era of metalwork.",
                    "rarity":0,
                    "sellPrice":0,
                    "buyPrice":0,
                    "carryLimit":0
                }
            },
            {
                "quantity":3,
                "item":{
                    "id":483,
                    "name":"High Commendation",
                    "description":"A ticket awarded for demonstrating tremendous courage. Collect them and something wonderful may happen!",
                    "rarity":0,
                    "sellPrice":0,
                    "buyPrice":0,
                    "carryLimit":0
                }
            },
            {
                "quantity":2,
                "item":{
                    "id":125,
                    "name":"Novacrystal",
                    "description":"High quality Lightcrystal. Highly coveted for its rarity, it's used for only the finest of metalworking.",
                    "rarity":0,
                    "sellPrice":0,
                    "buyPrice":0,
                    "carryLimit":0
                }
            },
            {
                "quantity":2,
                "item":{
                    "id":173,
                    "name":"Wyvern Gem",
                    "description":"Throughout a wyvern's life, impurities gather and form this peculiar gem.",
                    "rarity":0,
                    "sellPrice":0,
                    "buyPrice":0,
                    "carryLimit":0
                }
            }
        ]
    }
}
```

**Comment by LartTyler** [comment]
Thanks for pointing this out. Looks like it's wrong on Kiranico (which is where all my data comes from), and I just didn't notice. I'll get that fixed ASAP.

**Comment by rjb8682** [comment]
No problem at all! I'll make sure to email you next time instead of making a GH issue as well. Didn't see your message at the bottom of the wiki page until after I posted the issue :P 

Otherwise, fantastic job on this, man! Love it

**Comment by LartTyler** [comment]
Glad you're enjoying the API! Opening an issue is actually the better way to report bugs or request changes, I need to update that footer.

**Comment by LartTyler** [comment]
This has been resolved with [v1.5.0](https://github.com/LartTyler/MHWDB-Docs/releases/tag/1.5.0).
